tests/phpunit/tests/image/editorGd.php

```diff
@@ -17,6 +17,8 @@ public function set_up() {
 		require_once ABSPATH . WPINC . '/class-wp-image-editor.php';
 		require_once ABSPATH . WPINC . '/class-wp-image-editor-gd.php';
 
+		remove_filter( 'image_editor_output_format', 'wp_default_image_output_mapping' );
```

Comment:
Two things here: First, see above, this needs to be cleaned up in a `tear_down` method. However, more importantly, I'm wondering, why do you here remove the default filter while in the other class above you add a `__return_empty_array` filter?

I would assume both do the same, so it probably doesn't matter, still I'd argue we should align on one way to use throughout. From my perspective adding the `__return_empty_array` filter is a bit cleaner than to remove the default core filter, as that way at least we don't _remove_ from core's behavior. But I guess really it doesn't matter which way we go - I was mostly curious whether using different ways was intentional on your end, and if so why.

Reply:
I started with the return empty array approach, then switched at some point, i thought i had switched them all but apparently missed one.  we can go with the `__return_empty_array` approach, that was my first choice anyway.